web/template/donor/your_details.gohtml

```diff
@@ -42,10 +43,14 @@
                         .Donor.Donor.FullName .Donor.CanChange true) }}
                 </dl>
 
-                {{ if not $hasNoIdentity }}
+                {{ if .Donor.IdentityUserData.Status.IsConfirmed }}
                     <p class="govuk-inset-text">{{ tr .App "someOfTheseDetailsCanNoLongerBeChanged" }}</p>
                 {{ end }}
 
+                {{ if $vouchInProgress }}
```

Comment:
I've added https://opgtransform.atlassian.net/browse/MLPAB-2778 to get some updated content.